### ISSUE TITLE
New Website Discussion

### ISSUE THREAD

**Issue by daminisatya** [FossClub/FossClub.github.io#6]
Open question
----------------

What do we want to see? 

* Blog posts
* Contributors
* Hall of fame
* Projects

**Comment by Pranavya** [comment]
Our website would be more vigorous if we include all these things in it.

**Comment by iamsusmitha** [comment]
Yeah, I agree with you @Pranavya We will include all those things @daminisatya

**Comment by kavithaenair** [comment]
@Pranavya @Sush97 any improvement ? I can see no discussion going on here neither PRs or issues.
